load_balancer/package.json

```diff
@@ -5,7 +5,8 @@
   "main": "load_balancer.js",
   "scripts": {
     "start": "node load_balancer.js",
-    "test": "mocha -u bdd -R spec ./tests/"
+    "test": "mocha -u bdd -R spec ./test/",
```

Comment:
Given that we are going to have unit, integration, function tests, I suggest having the following structure for test command.
```
    "test": "bash test.sh"
```
invocation: `$TEST_TYPE=UNIT npm test`
The valid TEST_TYPE values are: UNIT, INTEGRATION, FUNCTION, THROUGHPUT

The `test.sh` shell script can have a case structure based detailed test script invocations.

For more details, please see [stackoverflow](https://stackoverflow.com/questions/11580961/sending-command-line-arguments-to-npm-script) and [tutorial](http://www.marcusoft.net/2015/08/npm-scripting-configs-and-arguments.html) pages.